indico.global has launched (#65)
indico/indico.github.io: _posts/2025-29-01-indico-global-launched.md

---
layout:     post
title:      "indico.global is live!"
date:       2025-29-01 17:00
author:     "Indico Team"
categories: indico global
---

> We are happy to announce that [indico.global](indico.global) is now available to

Reviewer: ```suggestion
We are happy to announce that [indico.global](https://indico.global) is now available to
```

I would remove the links from the other occurrences below, one is probably enough...

research centers, laboratories, universities, and educational institutions for

> them to organize and host meetings, lectures and conferences, who do not have

Reviewer: ```suggestion
them to organize and host meetings, lectures and conferences, if they do not have
```

the means to operate a dedicated Indico server. The service remains free and
accessible within our fair use policy, ensuring that every researcher has the
tools they need to manage their events.

All events from the old “Global Indico” category on

> [indico.cern.ch](indico.cern.ch) have been successfully migrated to the new

Reviewer: ```suggestion
[indico.cern.ch](https://indico.cern.ch) have been successfully migrated to the new
```

Reviewer: likewise below for the indico.global links
